packages/api/src/webhooks/cannon.ts

```diff
@@ -464,6 +440,36 @@ export default class WebhookCannon {
     }
   }
 
+  private async checkIsLocalIp(url: string, isAdmin: boolean) {
```

Comment:
I wonder if it's possible and if it makes sense to add some unit tests for this one. Wdyt?

Reply:
Yeah good call. Done!